uni/lib/model/entities/library_occupation.dart

```diff
@@ -0,0 +1,75 @@
+/// Overall occupation of the library
+class LibraryOccupation {
+  int occupation = 0;
+  int capacity = 0;
```

Comment:
Can this be final?

Reply:
I don't think so. Those are just default values. Both are changed when floors are added.